core/src/main/java/com/yahoo/ycsb/workloads/CoreWorkload.java

```diff
@@ -505,7 +525,7 @@ public String buildKeyName(long keynum) {
     if (!orderedinserts) {
       keynum = Utils.hash(keynum);
     }
-    return "user" + keynum;
+	return "user"+String.format("%0"+zeropadding+"d",keynum);
```

Comment:
I think it is fine for now but if you want to dig more, check out https://issues.apache.org/jira/browse/HBASE-15569. Manually doing the substitution was more performant.